v1/pipeline/steps/cd/k8s-rolling-step-node.yaml

```diff
@@ -1,55 +1,40 @@
 title: K8sRollingStepNode
 type: object
 required:
-- identifier
-- name
-- spec
-- type
+  - spec
+  - type
 properties:
-  description:
+  desc:
     type: string
-    desc: This is the description for K8sRollingStepNode
+    desc: This is the description for k8s-rolling-step
```

Comment:
this used to be description right?

Reply:
Or we dont need this. 
I remember @archit-harness suggested that we only need only this in cruds

Reply:
Yes, earlier it was description. But we use the `desc` instead of description in V1 schema.
I think we can have the desc in the step. Users can add it if they want to. Although its not required field.